ci/build_cpp.sh

```diff
@@ -25,10 +25,15 @@ export RAPIDS_ARTIFACTS_DIR
 # populates `RATTLER_CHANNELS` array and `RATTLER_ARGS` array
 source rapids-rattler-channel-string
 
+# Use sed to replace the RAPIDS_CUDA_VERSION string in the variants file with
+# the value of that variable in this shell.
+sed -i "s/RAPIDS_CUDA_VERSION/${RAPIDS_CUDA_VERSION}/g" "conda/recipes/libkvikio/variants_$(arch).yaml"
```

Comment:
One thing that has worked well before, is to [output this to a variant file]( https://github.com/rapidsai/pynvjitlink/blob/68e63ae785c4b2b32497ec2e091988761d64bd6b/ci/build_conda.sh#L20-L26 )

It does a similar thing admittedly. Though it can be a bit clearer about what is happening. That could just be personal preference though

Reply:
I like that alternative! I agree it is clearer.

Reply:
I considered this possibility as well. However, in this instance it's not quite so simple because we need other things to be encoded in the variants as well. The behavior on x86 and arm is different because the former is a single variant while the latter requires two, so we would need to write out variants conditionally based on that. Moreover, for arm we need to use a different compiler version for each CUDA version, so it's a longer file. If you both prefer that I can switch to it, we just wind up with two fairly long here-docs instead of this command. The flip side is that we can drop the committed variants files if we choose to go that route.